readme.md

```diff
@@ -46,17 +45,73 @@ The command is registered with laravel as ```garbageman:purge```.  You can run i
 1. from the command line ```php artisan garbageman:purge;```
 2. via scheduled task.
 
-To automatically run the script as a scheduled job, then add the following to the schedule method of App\Console\Kernel.php:
+To automatically run the script as a scheduled job, then add the following to the schedule method of 
+`App\Console\Kernel.php`:
 
 ```php
-$schedule->command('garbageman:purge')
-         ->daily();
+    $schedule->command('garbageman:purge')
+             ->daily();
 ```
 
-You can use whatever schedule that you need to keep the records purged out. Just review the list at http://laravel.com/docs/master/scheduling#schedule-frequency-options. 
+You can use whatever schedule that you need to keep the records purged out. Just review the list at 
+[http://laravel.com/docs/master/scheduling#schedule-frequency-options](http://laravel.com/docs/master/scheduling#schedule-frequency-options).
 
-You can also use any of the advanced configuration options of the task scheduler like "Task Output" or "Task Hooks" as listed on the [Laravel documentation](http://laravel.com/docs/master/scheduling).
+You can also use any of the advanced configuration options of the task scheduler like "Task Output" or "Task Hooks" as 
+listed on the [Laravel documentation](http://laravel.com/docs/master/scheduling).
 
-## Configure models to cleanup
+## Configuration
 
-During the install process `config/garbageman.php` as copied to the project. That file is fully documented.
+During the install process `config/garbageman.php` is copied to the config directory of the project. That file is fully documented.  You will need to make the changes to that file to suit your needs.
+
+There are 3 main configuration items...
+
+1. Fire purge events - Fire events on purge of each record.
+2. Logging level - Level to log.
+3. Schedule - Models & number of days to allow the soft deleted record to stay.
+
+### Fire purge events (fire\_purge\_events)
+Allow hook into the purge of each record by throwing events before & after deleting of each record. There are 2 events thrown:
+
+* garbageman.purging:\<full/model/name\>
+* garbageman.purged:\<full/model/name\>
+
+The model is passed with each of the events. The "purging" event is thrown just *before* the actual delete & "purged" is thrown just *after* the actual delete.
+
+This is an expensive operation as it requires a SQL command for each record to delete so that the record can be thrown with the events. Therefore, unless you need to catch the events to preform some other action, leave this false to allow all records per model to get deleted with a single SQL call.
```

Comment:
@jimmypuckett Instead of "leave this false", do you mean to leave it blank/null?